### PR TITLE
[MIG] account_invoice_margin_sale_delivered_sync: Migration to 13.0 

### PR DESCRIPTION
**Pull request by chienandalu**
Keep purchase price delivered between sale order line and invoice lines

cc @Tecnativa TT32680

ping @sergio-teruel @carlosdauden

### CODE REVIEW

File: account_invoice_margin_sale_delivered_sync/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    purchase_price = fields.Float(
+        string="Purchase Price",
+        compute="_compute_purchase_price",
+        store=True,
+        readonly=False,
+        # HACK: Allow to write this field
+        # Remove in v13
+        inverse=lambda self: self,
+    )
```
Comment (sergio-teruel) [inline account_invoice_margin_sale_delivered_sync/models/account_move.py]: @chienandalu remove the inverse attribute

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
Travis fails

File: account_invoice_margin_sale_delivered_sync/models/account_move.py (hunk 1 above)
Comment (chienandalu) [reply account_invoice_margin_sale_delivered_sync/models/account_move.py]: :+1:

**Comment by chienandalu** [comment]
Travis green at last

**Review by sergio-teruel — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**